Tests/Common/APITest.swift

```diff
@@ -37,6 +37,7 @@ class CommonAPITest: UnitTest {
         // Factory method should allow nil values for `SdkConfig` to enable fallback to defaults.
         _ = SdkConfig.Factory.create(logLevel: nil)
         // Wrapper SDKs should be able to create a `SdkConfig` from a dictionary.
-        _ = SdkConfig.Factory.create(from: [:])
+        let configOptions: [String: Any] = [:]
```

Comment:
While writing tests for the TODOs, I realized the type was unspecified in these tests. So, I've updated them to ensure the dictionary type is specific to what we expect, rather than generic.